### ISSUE TITLE
remove freshen pass?

### ISSUE THREAD

**Issue by stchang** [vishesh/racketscript#161]
Performance of `let` seems to have improved: https://jsperf.com/let-vs-var-performance/90

Could RacketScript directly translate `let` and remove the freshen phase? Were there other purposes for freshening?

This would also improve the readability of the generated code.

**Comment by vishesh** [comment]
Yeah, I think `let` can replace `var` now. I don't recall the reason for using `var`. Also, I think , JS engine can also potentially do some memory optimizations because lifetime of `let` is more constrained than `var`.

**Comment by stchang** [comment]
Cool. I think this would simplify other translations too, since a lot of times `let`s get flattened into multiple `var`s, which then require subsequent transformations like return lifting
